spec/Features/food_index_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe 'Food Index Page' do
+  before(:each) do
+    @user = create(:user)
+    @user.confirm
+    @food = create(:food, user_id: @user.id)
+    login_as(@user, scope: :user)
+  end
+
+  it 'can see the add food button' do
```

Comment:
Great job team 🤝
- You have done a great job implementing your tests. However, some of your test cases are still failing the checks. Kindly fix these tests and ensure all test cases are passing.
![fail-test](https://github.com/Chimoneg27/Rails-Recipe-App/assets/63917725/fc019cd1-f9a3-4706-afbb-3da7eb0faa88)